project/CodegenPlugin.scala

```diff
@@ -33,10 +34,12 @@ object CodegenPlugin extends AutoPlugin {
 
   val RInstallTag = Tags.Tag("rInstall")
   val TestGenTag = Tags.Tag("testGen")
+  val DotnetTestGenTag = Tags.Tag("dotnetTestGen")
```

Comment:
We should make a pyTestGen as well and just unify them with testGen so that testGen does it for all languages which I think more closely aligns with the other commands

Reply:
Done!